mlflow/sentence_transformers/__init__.py

```diff
@@ -221,6 +221,29 @@ def log_model(
     Log a ``sentence_transformers`` model as an MLflow artifact for the current run.
 
     :param model: A trained ``sentence-transformers`` model.
+
```

Comment:
we'll need the rst declaration to notify page rendering to display this as a text block. 

``` rst

.. code-block:: python

```

Reply:
done!